### PR TITLE
Found day 39 to not have annotations etc. Went ahead and started there.

### PR DESCRIPTION
**Pull request by effect0r**
Check over it to make sure everythings correct. The timing may be +- 0.5 sec.

### CODE REVIEW

File: src/documents/videos/game-architecture/day039.html.md [hunk 1]
```diff
@@ -1,4 +1,51 @@
...
+    "0:37:10": "Fixing the movement"
+    "0:40:25": "Moving the camera to the player in Z"
+    "0:41:09": "Keeping the camera locked to the player"
+    "0:47:25": "Fixing left and top clipping"
+    "0:48:15": "Explanation of how to fix clipping"
+    "0:51:40": "Checking the framerate"
+    "0:52:38": "Fixing live code loading"
+    "0:59:29": "Testing if hotloading is still functioning"
+    "1:00:35": "Q&A"
+    "1:01:46": "Can you do that live code with a fucntion call? That could just be a poke. -Geneticspecies"
```
Comment (effect0r) [inline src/documents/videos/game-architecture/day039.html.md]: Grumble. Typo.

File: src/documents/videos/game-architecture/day039.html.md [hunk 2]
```diff
@@ -1,4 +1,51 @@
...
+    "1:02:05": "Enumerations. Am I correct to say you've not used them in Handmade Hero yet? I'm guessing this is simply because it's to early to know what will eventually require enumerations and not some other reason."
+    "1:02:38": "Why not layer the individual bodyparts into a single bitmap that way you only have to draw one image instead of three?"
+    "1:03:20": "Are you wearing a blue cape like the Handmade Hero today?"
+    "1:03:31": "I just wanted to point out that there is a magnifying tool in windows that came with aero"
+    "1:05:09": "Don't you want to test the hotloading? -Andysz"
+    "1:05:39": "HAHA YOU SWORE! -Garlandobloom"
+    "1:06:07": "Can you add a <stddef.h> include to handmadehero.h?"
```
Comment (effect0r) [inline src/documents/videos/game-architecture/day039.html.md]: handmade_hero.h, not handmadehero.h

File: src/documents/videos/game-architecture/day039.html.md [hunk 3]
```diff
@@ -1,4 +1,51 @@
...
+    "1:07:56": "Why does the guy have no feet?"
+    "1:08:17": "How do you know which compiler options, like clr and standard headers, you want to use?
```
Comment (effect0r) [inline src/documents/videos/game-architecture/day039.html.md]: Needs a quote at the end.

**Comment by abnercoimbre** [comment]
@effect0r +1

**Pull request merged**